setup.py

```diff
@@ -11,7 +11,7 @@
     name='openwpm-utils',
     license='MPL 2.0',
     url='https://github.com/mozilla/openwpm-utils',
-    version='0.2.0',
+    version='0.10.0',
```

Comment:
We shouldn't have this match OpenWPM's version. OpenWPM will increment the version number with each new FF release. This library won't.